CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+**Internal**
```

Comment:
You have to prefix this with `## Unreleased`, otherwise the auto-changelog will not recognize this.

I recently fixed the auto-changelog after the fact: https://github.com/getsentry/symbolic/commit/6970909942af44aaadd2fc4d8c91133d6b3dd405